File: examples/polyglot/stable_failure.py
from datetime import datetime, timezone
import os
import sys


timestamp = datetime.now(timezone.utc).isoformat().replace("+00:00", "Z")
print(f"[{timestamp}] RuntimeError: checkout total mismatch", file=sys.stderr)
print(f"pid={os.getpid()}", file=sys.stderr)
sys.exit(1)

File: examples/polyglot/stable_failure.go
package main

import (
	"fmt"
	"os"
	"time"
)

func main() {
	fmt.Fprintf(os.Stderr, "[%s] Error: checkout total mismatch\n", time.Now().UTC().Format(time.RFC3339Nano))
	fmt.Fprintf(os.Stderr, "pid=%d\n", os.Getpid())
	os.Exit(1)
}
